PREP_SYSTEM_FOR_DIETPI.sh

```diff
@@ -566,6 +557,35 @@ _EOF_
 		aPACKAGES_REQUIRED_INSTALL+=('device-tree-compiler') #Kern
 
 	fi
+	
+	G_DIETPI-NOTIFY 2 "WiFi selection"
+
+	WHIP_TITLE='WiFi required?'
+	WHIP_DESC='Please select an option'
+	WHIP_DEFAULT_ITEM=1
+	WHIP_MENU_ARRAY=(
+
+		'0' "I don't require WiFi, do not install."
+		'1' 'I require WiFi functionality, keep/install related packages.'
+
+	)
+
+	Run_Whiptail
+	if (( $WHIP_RETURN_VALUE == 1 )); then
+
+		G_DIETPI-NOTIFY 2 "Marking WiFi as needed"
+
+		aPACKAGES_REQUIRED_INSTALL+=('crda')			# WiFi related
+		aPACKAGES_REQUIRED_INSTALL+=('firmware-atheros')	# WiFi dongle firmware
+		aPACKAGES_REQUIRED_INSTALL+=('firmware-brcm80211')	# WiFi dongle firmware
+		aPACKAGES_REQUIRED_INSTALL+=('firmware-ralink')		# WiFi dongle firmware
+		aPACKAGES_REQUIRED_INSTALL+=('firmware-realtek')	# WiFi dongle firmware
+		aPACKAGES_REQUIRED_INSTALL+=('iw')			# WiFi related
+		aPACKAGES_REQUIRED_INSTALL+=('rfkill')	 		# WiFi related: Used by some onboard WiFi chipsets
+		aPACKAGES_REQUIRED_INSTALL+=('wireless-tools')		# WiFi related
+		aPACKAGES_REQUIRED_INSTALL+=('wpasupplicant')		# WiFi related
+
+	fi
 
```

Comment:
We need to delete the array here, just incase we need to init it again, with different entries:
```
unset WHIP_MENU_ARRAY
```